### ISSUE TITLE
Contribution Icon appears blurry

### ISSUE THREAD

**Issue by iamareebjamal** [RoboClub-core/roboclub-amu#5]
We are using vector assets, and yet icon appears blurry
![device-2017-04-07-155938](https://cloud.githubusercontent.com/assets/3874064/24796605/3a3e6ea0-1bab-11e7-945d-2fdee79d04a7.png)

**Comment by RJ722** [comment]
Does it appears blurry for the smaller ones too?

**Comment by iamareebjamal** [comment]
No, small ones are fine but I guess that's because they aren't big enough :wink:

**Comment by iamareebjamal** [comment]
Ironically appears crisp on 4.4 which has no support of vector drawables :stuck_out_tongue_winking_eye: 

![device-2017-04-08-233400](https://cloud.githubusercontent.com/assets/3874064/24831235/f1836832-1cb3-11e7-89d9-75efbf20f370.png)

**Comment by vennelakoilada362** [comment]
i want to work on this issue, please assign it to me

**Comment by RJ722** [comment]
@vennelakoilada362 sure, go ahead!

Do you have a fix in mind? I seem to have no idea what the problem here seems to be.
